crates/pathfinder/src/config.rs

```diff
@@ -111,6 +121,10 @@ impl Configuration {
         // users config filepath (if supplied).
         let (cfg_filepath, cli_cfg) = cli::parse_cmd_line();
 
+        if cfg_filepath.is_some() {
+            tracing::warn!("'--config' is deprecated. Consider using script files to retain the same functionality.");
+        }
```

Comment:
This config stuff is getting messy. I would like to move to only using clap to handle conflicting options etc in one place only. So I think we should sunset support for config files?

Reply:
What's a `script file`?

But I'm all for deprecating and removing config file support and just use clap.

Reply:
Hmm. I meant like a bash script.

Reply:
Maybe something like _Consider using command-line arguments or environment variables_ would be better? It's more explicit.

Reply:
But neither of those replace the functionality of a configuration file